app/pages/homepage/ui.test.js

```diff
@@ -54,47 +63,98 @@ test('should navigate to home page header banner', async (t) => {
 });
 
 test('when user is not authenticated - should display the login link', async (t) => {
-  await pageSetup(t);
+  await pageSetup({ t });
 
   const loginComponent = Selector('[data-test-id="login-logout-component"] a');
   await t
     .expect(await extractInnerText(loginComponent)).eql('Log in');
 });
 
 test('when user is not authenticated - should navigate to the identity server login page when clicking the login link', async (t) => {
-  await pageSetup(t);
+  await pageSetup({ t });
 
   nock('http://identity-server')
     .get('/login')
     .reply(200);
 
-  const getLocation = ClientFunction(() => document.location.href);
   const loginComponent = Selector('[data-test-id="login-logout-component"] a');
   await t
     .click(loginComponent)
     .expect(getLocation()).eql('http://identity-server/login');
 });
 
 test('when user is authenticated - should display the logout link', async (t) => {
-  await pageSetup(t, true);
+  await pageSetup({ t, withAuth: true, withLoggedInCookies: true });
 
   const logoutComponent = Selector('[data-test-id="login-logout-component"] a');
   await t
     .expect(await extractInnerText(logoutComponent)).eql('Log out');
 });
 
+test('when user is authenticated - should display log in text once log out link is clicked', async (t) => {
+  await pageSetup({ t, withAuth: true, withLoggedInCookies: true });
+
+  const logoutComponent = Selector('[data-test-id="login-logout-component"] a');
+  await t
+    .expect(logoutComponent.exists).eql(true)
+    .click(logoutComponent)
+    .expect(Selector('[data-test-id="login-logout-component"] a').withText('Log in').exists).eql(true)
```

Comment:
can we just check the inner text of the `logoutComponent` is `Log in`.

Reply:
This is checking that?